src/UnisonShare/Project.elm

```diff
@@ -247,7 +300,8 @@ decodeDetails =
         |> optional "numActiveContributions" int 0
         |> optional "numOpenTickets" int 0
         |> required "releaseDownloads" ReleaseDownloads.decode
-        |> optional "isFaved" decodeIsFaved Unknown
+        |> optional "isFaved" decodeIsFaved FavUnknown
+        |> optional "isSubscribed" decodeIsSubscribed SubUnknown
```

Comment:
@ChrisPenner this expects a new `isSubscribed` field on the project details endpoint.